frb_codegen/src/config.rs

```diff
@@ -51,15 +52,15 @@ pub struct RawOpts {
     pub llvm_compiler_opts: Option<String>,
     /// Path to root of Dart project, otherwise inferred from --dart-output
     #[structopt(long)]
-    pub dart_root: Option<String>,
+    pub dart_root: Option<Vec<String>>,
```

Comment:
Do we really need multiple dart roots? i.e. users will maybe only generate multiple dart files into *one* dart package

Reply:
> > For flag class_names and rust-output, if there are multiple blocks, users have to submit all data for each block. This is essential for compilation and readability.
> 
> What is "all data for each block"? Sorry do not quite get it

For example,  with 2 rust blocks, if I do it like this, without specific any data for `rust-output`:
```
gen:
    # union version
    export REPO_DIR="$PWD"; cd /; flutter_rust_bridge_codegen {{llvm_path}} \
        --rust-input "$REPO_DIR/native/src/rule_api.rs" "$REPO_DIR/native/src/ucci_api.rs" \
        --dart-output "$REPO_DIR/lib/gened_rule_api.dart" "$REPO_DIR/lib/gened_ucci_api.dart" \
        # --rust-output "$REPO_DIR/native/src/gened_rule_api.rs" "$REPO_DIR/native/src/gened_ucci_api.rs" \
        --class-name RuleApi UcciApi \
        # --exclude-sync-execution-mode-utility false true
```
Then, frb would throw panic ` 'for more than 1 rust blocks, please specify each rust output path clearly with flag "rust-output"'`. The reason, in detail,  if no any data on `rust-output`, then code would generated in one rust file, which I found not compilable finally, and also this violate the original intension for using multiple blocks. Sames things for flag `class-name`.

Reply:
According to the original code, If no `dart_root` provided, then the final `dart_root` would be generated by `fallback_dart_root`,which is in relation with `dart_output_path`. And `dart_output_path` would be different for each rust block if `dart-output` is specifed(just like my example).   

Therefore, I think `dart_root` need to be a `vec`.

Reply:
Sounds ok.